moveit_core/profiler/include/moveit/profiler/profiler.h

```diff
@@ -60,6 +60,9 @@
 #include <boost/thread.hpp>
 #include <boost/noncopyable.hpp>
 #include <boost/date_time/posix_time/posix_time.hpp>
+#include <rcutils/logging_macros.h>
+
+#define ROS_INFO_STREAM_NAMED RCUTILS_LOG_INFO
```

Comment:
consolidate per this comment:
https://github.com/ros-planning/moveit2/pull/10#discussion_r258226469

Reply:
Pushed to https://github.com/ros-planning/moveit2/issues/21

Reply:
I don't understand what you mean by "pushed to an issue", we don't want to introduce hacks to the code. Since the changes to profiler.h are all logging-related, perhaps you should remove them from this PR until a unified logging approach is decided on

Reply:
@davetcoleman, I meant to push the discussion to a single thread. This same issue has been highlighted across several of the existing PRs. My suggestion is to agree on how to proceed in the issues and then we'll apply such changes to the different branches.

Reply:
agreed, so can we remove the change from this PR so it can be merged in now?

Reply:
https://github.com/ros-planning/moveit2/pull/9/commits/6f9d7b3ef5e15d5541c67648dda7b27f7f6bbf41

Reply:
we decided to not use this ``#define ROS_INFO_STREAM_NAMED RCUTILS_LOG_INFO`` approach, please remove

Reply:
My bad, please see 73e5a99677af2cecd29eee6256531222c0f2b62e